src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -161,9 +161,43 @@ button.NormalAnnouncementBubble--deleteMenuItem {
   }
 }
 
+.NormalAnnouncementBubble--translatedLanguages--tooltip {
+  padding-left: @size_xs;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--icon {
```

Comment:
Is there a way to make an SVG colorable by CSS? Right now it's just always gray. I'm going to check with Nick if we want it to change blue on hover as well

If so, not sure what the easiest way to check hover state is in the .tsx file would be if that's possible, otherwise maybe have to add the icon as a component like CheckMark (src/Checkbox/CheckMark.tsx) instead of as an svg file?

Reply:
Looks like I could do something like what Brian has in his [PR](https://github.com/Clever/components/pull/718/files) actually, using a div and `onMouseOver`, for the first option in the second paragraph above. Curious which approach feels better to you!